File: filesystem-processor/src/main/java/com/atlassian/migration/datacenter/fs/processor/FilesystemProcessorApplication.java
package com.atlassian.migration.datacenter.fs.processor;

import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "com.atlassian.migration.datacenter")
public class FilesystemProcessorApplication implements CommandLineRunner {

    public static void main(String[] args) {
        SpringApplication.run(FilesystemProcessorApplication.class, args);
    }

    @Override
    public void run(String... args) throws Exception {
    }
}

File: filesystem-processor/src/main/kotlin/com/atlassian/migration/datacenter/fs/processor/FileSystemProcessorApplication.kt
package com.atlassian.migration.datacenter.fs.processor

import org.springframework.boot.Banner
import org.springframework.boot.WebApplicationType
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.atlassian.migration.datacenter"])
open class FileSystemProcessorApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            runApplication<FileSystemProcessorApplication>(*args) {
                this.webApplicationType = WebApplicationType.NONE
                this.setBannerMode(Banner.Mode.OFF)
            }
        }
    }


}